docker/service-base-requirements.txt

```diff
@@ -1 +1,3 @@
-google-cloud-profiler==4.0.0
+# https://github.com/sass/libsass-python/issues/413
+google-cloud-profiler<4.0.0
+protobuf<=3.20.0
```

Comment:
protobuf is already in the docker pinned requirements as a transitive dependency, so I'm guessing this will downgrade it. That should probably be pinned in `docker/requirements.txt` instead.